autoload/tagtracks.vim

```diff
@@ -79,12 +74,22 @@ function tagtracks#StartTagTracks()
   endif
   " window id whose tagstack we want to display
   const tagtracks_id = win_getid()
-  new
+
+  " make HUD 1/4 the height of current window instead of the default 1/2
+  const win_height = winheight(0) / 4
+  execute win_height.'new'
+
   setlocal nonumber
+  setlocal nowrap
   setlocal buftype=nofile
   setlocal bufhidden=wipe
   setlocal noswapfile
   setlocal nobuflisted
+
+  " set colors in display to mirror :tags
+  syntax match Title /\vItem\s+Tag\s+Origin/
+  syntax match Directory /\v\d+\s+\S+\s+\S+:\d+\s+\zs.*$/
+
```

Comment:
Let's delete these empty lines too, so the "new window" bit is all grouped together.

Reply:
And done